src/django_components/template_loader.py

```diff
@@ -56,18 +78,19 @@ def get_dirs(self) -> List[Path]:
             for app_config in apps.get_app_configs():
                 if not app_config.path:
                     continue
-                if not Path(component_dir).is_dir():
-                    continue
 
-                if path_is_relative_to(component_dir, app_config.path):
+                # NOTE: This applies only to absolute paths
+                if path_is_relative_to(component_dir, app_config.path) and Path(component_dir).is_dir():
                     curr_directories.add(Path(component_dir).resolve())
```

Comment:
Thinking out loud as I'm trying to make sense of this - If we get here to line 84, that means that the current dir from `STATICFILES_DIRS` points to a place WITHIN the current app.

So if the Django project root path is `/path/to/project`, and app's path is `/path/to/project/my_app/`, then `component_dir` also starts with `/path/to/project/my_app/`. So e.g. `/path/to/project/my_app/my_comps/`, in which case `path_is_relative_to()` would return `./my_comps/`.

If all this is true, then we know that the current path from STATICFILES_DIRS is an absolute path, and we can end the loop early, as the rest of the code makes sense only if the path is relative.